ci/build.ps1

```diff
@@ -94,6 +94,28 @@ npm.cmd install -g neovim
 Get-Command -CommandType Application neovim-node-host.cmd
 npm.cmd link neovim
 
+#npm.cmd install -g tree-sitter-cli
+#npm.cmd link tree-sitter-cli
+
+mkdir c:\treesitter
+$env:TREE_SITTER_DIR = "c:\treesitter"
+#$env:PATH = "c:\treesitter;$env:PATH"
+$client = new-object System.Net.WebClient
+cd c:\treesitter
```

Comment:
creating things in root (`c:\`) is bad practice and might not work on non-CI machines (depending on user permissions). Although build.ps1 isn't fully usable for non-CI (yet), would be good to avoid making it less so. Could use `build/` or `.deps/` .

Reply:
I assumed a file in `ci/` is for, well, CI and not a generic build script or something. But I will find some more reasonable path.

Reply:
Some programs rely on environment variables for installation paths without asking for elevated rights, for instance `%APPDATA%`. Maybe [%TMP% or %TEMP%](https://superuser.com/questions/709953/temp-vs-tmp-in-environment-variables) are set at the time the script runs and you could use it.